iOS/RocketApp.xcodeproj/xcshareddata/xcschemes/RocketApp.xcscheme

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "1420"
+   version = "1.7">
+   <BuildAction
+      parallelizeBuildables = "YES"
+      buildImplicitDependencies = "YES">
+      <PreActions>
+         <ExecutionAction
+            ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
+            <ActionContent
+               title = "BuildKMM"
+               scriptText = "cd &quot;$PROJECT_DIR&quot;&#10;./scripts/build_xcframework_script.sh DEBUG&#10;">
```

Comment:
hardcoded DEBUG. As I'm not familiar with Xcode development, is there a way to configure depending on `buildConfiguration = "Debug"` or similar?

Reply:
Xcode has schemes that you can set for different development types - AdHoc, Release, Debug etc. Right now the only scheme we have on iOS is Debug. Due to absence of pipelines ( CI / CD ) its not that necesarry to add more schemes.